src/lambdas/auditMessageFirehoseReingest/getAuditEventsFromS3Object.ts

```diff
@@ -0,0 +1,30 @@
+import { Readable } from 'node:stream'
+import { constants, createGunzip } from 'node:zlib'
+import { getS3ObjectAsStream } from '../../services/s3/getS3ObjectAsStream'
+import { AuditEvent } from '../../types/auditEvent'
+import { base64ToObject } from '../../utils/helpers/base64ToObject'
+import { readableToString } from '../../utils/helpers/readableToString'
+
+export const getAuditEventsFromS3Object = async (
+  bucketName: string,
+  key: string
+) => {
+  const contents = await getS3ObjectAsStream(bucketName, key)
+
+  return await getAuditEvents(contents)
```

Comment:
I'm never quite sure what `return await` will do, but I think it indicates that you should probably just be returning the promise. Would be good to add return types for your functions to make it clearer what's actually going on here as well.

Reply:
Fixed. Don't need the `await` either!